### PR TITLE
Add Explore and Dashoboard page and make token add function

### PR DESCRIPTION
**Pull request by toshiakitakase**
Explore page
- Add token
- routing to /explore/:tokenAddress

Dashboard page
- Add token
- routing to /dashboard/:tokenAddress

URL:
/
/dashboard
/explore

### CODE REVIEW

File: frontend/packages/react-app/src/components/molecules/TokenListItem/index.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  name: "Sample Token",
+  address: "0x",
+};
+
+export const Explore = Template.bind({});
+Explore.args = {
+  color: "itred",
+  name: "Sample Token",
+  address: "0x",
+};
```
[the remark this answers is not in the document]
Comment (wild-mouse) [reply frontend/packages/react-app/src/components/molecules/TokenListItem/index.stories.tsx:30]: Seems `type` property is lacked. 
```
export const Dashboard = Template.bind({});
Dashboard.args = {
  type: "dashboard",
  color: "itblue",
  name: "Sample Token",
  address: "0x",
}
```

File: frontend/packages/react-app/src/components/atoms/AddNewToken/index.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { Box, Link } from "rimble-ui";
+
+export interface AddNewTokenProps {
+  color: string;
+  dispatch({}: object): void;
```
[the remark this answers is not in the document]
Comment (toshiakitakase) [reply frontend/packages/react-app/src/components/atoms/AddNewToken/index.tsx]: Fixed

File: frontend/packages/react-app/src/components/molecules/TokenListItem/index.stories.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (toshiakitakase) [reply frontend/packages/react-app/src/components/molecules/TokenListItem/index.stories.tsx]: Fixed

**Review by wild-mouse — APPROVED** [review]
LGTM 🏃‍♀️

**Pull request merged**